code/style.css

```diff
@@ -1,3 +1,232 @@
+* {
+  box-sizing: border-box;
+  text-align: center;
+}
+/* variables to style the dark mode, add these in another variable in dark mode and add those to the selectors */
+:root {
+  --borderColor: 2px solid white;
+  --linkColor: white;
+  --usernameColor: white;
+}
+
 body {
-  background: #FFECE9;
-}
\ No newline at end of file
+  background: #f9f3ec;
+  font-family: "Stick No Bills", sans-serif;
+  letter-spacing: 2px;
+  margin: 0;
+  transition: 0.6s ease, color 1s ease;
+}
+
+.dark-mode {
+  background: rgba(0, 0, 0, 0.8);
+  color: white;
+  --borderImgColor: var(--borderColor);
+  --colorOfLink: var(--linkColor);
+  --colorOfUsername: var(--usernameColor);
+}
+
+.toggleBtn {
+  font-family: "Stick No Bills", sans-serif;
+  border-radius: 20px;
+  padding: 8px;
+  cursor: pointer;
+  font-size: 20px;
+}
+
+.toggleBtn:hover {
+  border: 2px solid #fff;
+  background: #5f939a;
+  color: #fff;
+}
+
+header {
+  background: #5f939a;
+  padding: 10px;
+  width: 100%;
+  margin-bottom: 30px;
+  text-transform: uppercase;
+}
+
+.username {
+  color: var(--colorOfUsername);
+  font-size: 24px;
+}
+
+.username-link {
+  text-decoration: none;
+  color: var(--colorOfLink);
+}
+
+.username-link:hover {
+  transition: all 1s;
+  background: #5f939a;
+  border-radius: 30px;
+  padding: 5px;
+}
+
+.repo-amount {
+  font-style: italic;
+}
+
+.repo-title {
+  text-transform: uppercase;
+}
+
+.push-date {
+  width: 100%;
+}
+
+.push-title {
+  font-weight: bold;
+  display: block;
+}
+
+.cards {
+  border: 2px solid #5f939a;
+  background-color: #dbd7d298;
+  display: grid;
+  width: 90%;
+  text-align: center;
+  margin: 10px auto;
+  border-radius: 20px;
+  padding: 0 5px;
+}
+
+.branch {
+  border: 2px solid black;
+  border-radius: 30px;
+  width: 30%;
+  margin: auto;
+  background: #5f939a;
+  color: white;
+}
+
+.grid-container {
+  margin: auto;
+}
+
+fieldset {
+  border: 4px solid;
+  border-color: #5f939a;
+  border-radius: 20px;
+  margin: 0 10px;
+}
+
+legend {
+  text-transform: uppercase;
+  font-size: 28px;
+}
+
+img {
+  border: var(--borderImgColor);
+  width: 70%;
+  border-radius: 50%;
+}
+
+.chart {
+  margin: 10px auto;
+  height: 200px;
+  width: 200px;
+}
+
+.links {
+  color: var(--colorOfLink);
+  width: 80%;
+  margin: auto;
+  text-decoration: none;
+  font-size: 12px;
+}
+
+.links:hover {
+  transition: all 1s;
+  background: #5f939a;
```

Comment:
I've seen that you have used this color a couple of times. Maybe can try to put this color in the root and store it in a variable? 🦊